Makefile

```diff
@@ -381,7 +381,7 @@ amazon-linux-sources.tgz:
 
 .amazon-linux-rpm-integrated-done: amazon-linux-sources.tgz
 	test -e SOURCES || ln -s . SOURCES
-	rpmbuild --define "%_topdir $(PWD)" -bb ecs-init.spec
+	rpmbuild --define "%_topdir $(PWD)" -bb ecs-agent.spec
```

Comment:
how was this working before if ecs-init.spec doesn't exist?

Reply:
actually that's a good point -- this was working because koji doesn't actually use this makefile so this target is more a reference to approximate what koji builds rather than the mechanism koji uses to build.